package.json

```diff
@@ -18,6 +18,17 @@
         "sort-specs": "node test/src/commands/sort-specs.js",
         "vscode:prepublish": "npm run build"
     },
+    "keywords": [
+        "C++",
+        "cpp",
+        "c",
+        "objective-c",
+        "objective-cpp",
+        "objective-c++",
+        "syntax",
+        "textmate",
+        "highlighting"
+    ],
```

Comment:
maybe added color(ing), a couple of other syntax extensions have that (tree sitter, c++ reloaded, yml, syntax highlighter)

Reply:
done 👍